src/main/kotlin/commands/BanCommand.kt

```diff
@@ -63,10 +66,10 @@ object BanCommand : Command("ban") {
                                 fixedDays,
                                 fixedReason
                             )
-                        } catch (e: MissingPermissionsException) {
+                        } catch (e: Exception) {
                             message.channel.send {
                                 embed {
-                                    title = "That user is protected, I can't do that."
+                                    title = "Failed to ban the user!"
```

Comment:
This should still be the same exception, just "That users role is higher then mine, I can't ban them"